<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stimuliController</class>
 <widget class="QWidget" name="stimuliController">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>537</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Cell Identifiers</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="txtCellIds">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>600</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Cell Selection URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtHBPURL">
       <property name="text">
        <string>hbp://</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConnectHBP">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDisconnectHBP">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Simulation URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtISCURL">
       <property name="text">
        <string>isc-nesteer://</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConnectISC">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDisconnectISC">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QTableView" name="tblGenerators"/>
     </item>
     <item>
      <widget class="QTableView" name="tblGeneratorProperties"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnInjectStimulus">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Inject Stimulus</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkIndividual">
         <property name="text">
          <string>Individually</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
